index.js

```diff
@@ -487,6 +497,11 @@ class MillerColumnsElement extends HTMLElement {
         // show this column in all it's glory
         item.classList.remove(collapseClass, narrowClass, mediumClass)
       }
+
+      // mark column with active item as active
```

Comment:
```suggestion
      // mark last column as active
```

Reply:
I was skeptical about naming it 'last column' as, depending on your interaction, it may not be the last one in the chain – but whatever makes more sense to everyone.

Reply:
"Last visible column" maybe?

Reply:
Updated with the suggestion.